mime-ballerina/Module.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+
+This module provides APIs to encapsulate multiple body parts such as attachments into a single message. 
```

Comment:
This module provides a set of APIs to work with messages which follow the Multipurpose Internet Mail Extensions (MIME) specification as specified in 
the [RFC 2045 standard](https://www.ietf.org/rfc/rfc2045.txt)

Reply:
@shafreenAnfar Shall we make the first line simple as it appears in the list view. Make consistent with other modules. WDYT?